src/openms/source/FORMAT/MascotRemoteQuery.cpp

```diff
@@ -50,10 +48,10 @@ namespace OpenMS
 
   MascotRemoteQuery::MascotRemoteQuery(QObject* parent) :
     QObject(parent),
-    DefaultParamHandler("MascotRemoteQuery"),
-    manager_(NULL),
-    reply_(NULL)
+    DefaultParamHandler("MascotRemoteQuery")
   {
+    http_ = new QHttp();
```

Comment:
I dont think that will compile ... afak there is no more QHttp ... 

I suggest to use the qt5 version with the Mascot 2.6 patch